### PR TITLE
add-docker-docs

### PR DESCRIPTION
**Pull request by Rawshansharma**
Fix issue #1888 

Add Docker documentation to learn docker

Screenshot 2023-10-08 174101

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @Rawshansharma. Welcome to LinksHub! :) We don't accept spam pull requests. In the future, please raise an issue and wait until it has been approved and assigned to you before you submit a pull request. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊

**Comment by Rawshansharma** [comment]
can i work on this issue ?

**Comment by CBID2** [comment]
> can i work on this issue ? 
> 
> 

Did you raise an issue like I instructed @Rawshansharma?

**Comment by CBID2** [comment]
Sorry @Rawshansharma. It's been assigned to someone else.

File: database/devops/docker.json [hunk 1]
```diff
@@ -12,5 +12,13 @@
         "url":"https://kodekloud.com/lessons/introduction-9/",
         "category":"devops",
         "subcategory":"docker"
+    },
+    {
+        "name":"Docker",
```
Comment (rupali-codes) [inline database/devops/docker.json:17]:
```suggestion
        "name":"Docker Documentation",
```

File: database/devops/docker.json [hunk 2]
```diff
@@ -12,5 +12,13 @@
         "url":"https://kodekloud.com/lessons/introduction-9/",
         "category":"devops",
         "subcategory":"docker"
+    },
+    {
+        "name":"Docker Documentation",
+        "description":"Docker is a popular platform for developing, shipping, and running applications in containers. The official Docker documentation serves as a comprehensive resource for users, offering detailed information, guidance, and best practices for effectively using Docker and its related tools.",
```
Comment (CBID2) [inline database/devops/docker.json:18]:
```suggestion
        "description":"The official Docker documentation serves as a comprehensive resource for users, offering detailed information, guidance, and best practices for effectively using the platform and its related tools.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Made some suggestions for improvement

File: database/devops/docker.json (hunk 2 above)
Comment (rupali-codes) [reply database/devops/docker.json:18]: The previous one looks more detailed i guess, no need to remove it :)

Comment (Anmol-Baranwal) [reply database/devops/docker.json:18]: Resolving the conversation.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Looks great to me!

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**